### PR TITLE
Generate `/docs/acs/systems/README.md` with `acs_system` resource and properties

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
closes https://github.com/seamapi/docs/issues/300

### CODE REVIEW

File: src/layouts/api-resource.hbs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{{#if description}}
+
+{{description}}
+{{/if}}
+
+Type: `{{jsonType}}`
+{{#if format}}
+Format: `{{format}}`
+{{/if}}
+---
```
Comment (razor-x) [inline src/layouts/api-resource.hbs:25]:
```suggestion

---
```

File: src/layouts/api-resource.hbs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+{{/each}}
+## Events
+
+{{~#each events}}
+### `{{name}}`
+
+{{description}}
+---
+
```
Comment (razor-x) [inline src/layouts/api-resource.hbs:46]:
```suggestion

{{#each endpoints}}
### [`{{path}}`]()

{{description}}

---

{{/each}}
## Events

{{~#each events}}
### `{{name}}`

{{description}}

---

```

File: src/layouts/api-resource.hbs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+{{/if}}
+
+---
+
+{{/each}}
+{{/each}}
+## Endpoints
+
+{{#each endpoints}}
+### [`{{path}}`]()
```
Comment (razor-x) [inline src/layouts/api-resource.hbs:33]:
```suggestion
### [`{{path}}`](./{{path}}.md)
```

File: src/lib/reference.ts [hunk 4]
```diff
@@ -26,14 +29,22 @@ export const reference = (
   }
 
   for (const route of metadata.routes ?? []) {
+    const k = `api${route.path}/README.md`
+    files[k] = {
+      contents: Buffer.from('\n'),
+    }
+    const file = files[k] as unknown as File
+    file.layout = 'api-resource.hbs'
```
Comment (razor-x) [inline src/lib/reference.ts:37]: Let's name this `api-route.hbs` now, and use `setApiRouteTemplateContext` etc.
```suggestion
    file.layout = 'api-route.hbs'
```

**Review by razor-x — APPROVED** [review]

File: src/lib/reference.ts (hunk 4 above)
Comment (andrii-balitskyi) [reply src/lib/reference.ts]: 6132664bbb9e1bd61c32e5c7108bd7b931e6ca65

**Pull request merged**